app/screens/snack_bar/index.tsx

```diff
@@ -45,6 +45,9 @@ const getStyleSheet = makeStyleSheetFromTheme((theme: Theme) => {
             position: 'absolute',
             bottom: 104,
         },
+        gestureRootChannelInfoAdjustment: {
+            bottom: 34,
```

Comment:
I believe this is wrong, 34 is the bottom space on iOS devices that have a safe area inset, so I need to ask, does this bottom apply to every device or only to those that have a safe area?

@matthewbirtch can you chip in here?
@cpoile in case this is required only for those devices that have a safe area you could then use something like this

```
const {bottom} = useSafeAreaInset();
const gestureRootChannelInfoAdjustment = useMemo(() => ({bottom: bottom > 0 ? bottom, undefined}), [bottom]);
```

or something along those lines.

Reply:
The toast should show 8 from the bottom for devices that do not have the iOS home indicator. Otherwise it should sit right at the top of the safe zone - 34 from the bottom

Reply:
<img width="480" alt="image" src="https://user-images.githubusercontent.com/2040554/198731801-8b6fd9c4-abe8-4316-b694-00d0996ec479.png">